### ISSUE TITLE
Status bar height doesn't work

### ISSUE THREAD

**Issue by mld9401** [siavash79/AOSPMods#35]
When i try to change status bar height nothing happens.

Device : POCO X3 NFC
ROM : PixelExperience ROM
Android Ver : 12.1/12L

**Comment by Underday** [comment]
yes i have the same device and in a different rom it still doesn't work

**Comment by EleoXDA** [comment]
Here is default ![Screenshot_20220527-184312.png](https://user-images.githubusercontent.com/27622683/170742747-ec0393cb-2fdb-4f97-9757-33c599c891d6.png)

Small ![Screenshot_20220527-184252.png](https://user-images.githubusercontent.com/27622683/170742825-c1ddb5cb-2e93-4cdc-ab48-18f341056c72.png)

And medium
![Screenshot_20220527-184301.png](https://user-images.githubusercontent.com/27622683/170742869-0566d48d-6720-4da6-bf16-7ef250fae657.png)

Can't you get these? If not, then please report android version, custom ROM name, device name and lsposed logs while you try to change these settings

**Comment by mld9401** [comment]
I think this module doesn't increase the status bar height more than default, it only makes it smaller. In my case i want to increase my status bar more height just like the default one on your first screenshot. Is it possible?

**Comment by EleoXDA** [comment]
> I think this module doesn't increase the status bar height more than default, it only makes it smaller. In my case i want to increase my status bar height just like the default one on your first screenshot. Is it possible?

- Did you read the telegram link to post that I have sent you? If not, read here:
![Screenshot_20220527-210817.png](https://user-images.githubusercontent.com/27622683/170775211-40e262c5-46d6-4bdb-8f68-f9fcd5770e0f.png)

As most other tweaks of this MOD, this tweak is also done with purely Google pixel devices in mind. And this tweak basically is to "shrink down" the statusbar height from default (huge) as you can see in my screenshots above. In reality, both medium and small statusbar heights are cutting the front camera in half, which doesn't look ok for me. But for everyone their own taste.

TLDR; no, this is only to shrink down the statusbar, not for extension.

So, this tweak works, but it is only just not what you want

**Comment by mld9401** [comment]
Aah Ok i understand now, sorry for the telegram message i didn't understand it well when i have read it the first time. Thanks for clarifying things for me.
